QUICK_START_GUIDE.md

```diff
@@ -446,7 +446,41 @@ optional arguments:
   --save-model-to SAVE_MODEL_TO
                         Location where openvino.zip will be stored.
 ```
+### ote explain
 
+`ote explain` runs explanation of a trained model on a particular dataset.
+
+With the `--help` command, you can list additional information, such as its parameters common to all model templates:
+command example:
+
+```
+ote explain ./external/model-preparation-algorithm/configs/classification/mobilenet_v3_large_1_cls_incr/template.yaml --help
+```
+
+output example:
+
+```
+usage: ote explain [-h] TEMPLATE
+                --input INPUT_IMAGE_ROOTS --output OUTPUT_FOLDER_ROOT
+                --load-weights WEIGHT_PATHS --xai-model XAI_MODEL_NAME
+
+positional arguments:
+  template
+  {params}              sub-command help
+    params              Hyper parameters defined in template file.
+
+optional arguments:
```

Comment:
Just a note. It is planned to support class discriminative explain algorithms by default as the next step. For now this falls out of scope of this PR. But for the future work, it is required to provide targets (target class(s) or target bbox(s) for classification and detection) as well.

Reply:
Yes, as we talked yesterday. For the current version, we only use whitebox models so that we don't need class id. But as you said, at the time we support black-box model, we can specify the class id, and we can accept the class ids from user and show only that ids'(single or multiple) saliency maps.

Reply:
Currently we support only xai algos that do not discriminate between classes. Thus, let's keep it as is. WB can be class discriminative. The same applies to BB